_config/config.yml

```diff
@@ -8,7 +8,6 @@ Silverstripe\SiteConfig\SiteConfig:
 Silverstripe\CMS\Model\SiteTree:
   extensions:
     - Sunnysideup\GetSiteControl\Extensions\PageExtension
-
-SilverStripe\CMS\Controllers\ContentController:
+Page:
   extensions:
```

Comment:
I dont understand why you dont add this to the ContentController. Adding an extension to SiteTree and to Page does not make sense to me, but I may be missing something.

Reply:
I tried with ContentController but it does not work. I didn't debug it enough to know why, sorry. Page should do nicely.

Reply:
Then please just move ALL code into the PageExtension. The way you have set it up is a bit weird.